ci/azure-Linux-35.yaml

```diff
@@ -0,0 +1,28 @@
+name: pandas
```

Comment:
Minor issue but should "Linux" be all lower case in the file name to be consistent with the windows yaml files?